Makefile

```diff
@@ -1,10 +1,22 @@
 #
 # Makefile - `gocat` (https://github.com/tuladhar/gocat)
 # 
-PROGRAM_NAME:=gocat
+PROGRAM_NAME=gocat
+INSTALL_PATH=/usr/local/bin
 
 build:
 	go build -o $(PROGRAM_NAME) src/main.go
 
-cleanup:
+install:
+	go build -o $(PROGRAM_NAME) src/main.go
```

Comment:
We can re-use the build target and remove this, like so:
```
install: build
```
Now, `build` will auto run before install.